boot/zephyr/Kconfig

```diff
@@ -376,81 +376,7 @@ config BOOT_FIH_PROFILE_HIGH
 
 endchoice
 
-config BOOT_USB_DFU
-	bool
-
-choice BOOT_USB_DFU_CHOICE
-	prompt "USB DFU"
-	default BOOT_USB_DFU_NO
-
-config BOOT_USB_DFU_NO
-	prompt "Disabled"
-
-config BOOT_USB_DFU_WAIT
-	bool "Wait for a prescribed duration to see if USB DFU is invoked"
-	select USB_DEVICE_STACK
-	select USB_DFU_CLASS
-	select IMG_MANAGER
-	select BOOT_USB_DFU
-	help
-	  If y, MCUboot waits for a prescribed duration of time to allow
-	  for USB DFU to be invoked. Please note DFU always updates the
-	  slot1 image.
-
-config BOOT_USB_DFU_GPIO
-	bool "Use GPIO to detect whether to trigger DFU mode"
-	select USB_DEVICE_STACK
```

Comment:
I advise you to do this as separate PR after https://github.com/mcu-tools/mcuboot/pull/1344 gets in, conflicts with moved files are not fun.